test/scrabble_test.rb

```diff
@@ -6,8 +6,40 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_an_entire_word
+    game = Scrabble.new
+    expected = 8
+    actual = game.score("hello")
+
+    assert_equal expected, actual
+  end
+
+  def test_it_can_score_a_blank_string
+    game = Scrabble.new
+    expected = 0
+    actual = game.score("")
+
+    assert_equal expected, actual
+  end
+
+  def test_it_can_score_nil
+    game = Scrabble.new
+    expected = 0
+    actual = game.score(nil)
+
+    assert_equal expected, actual
+  end
+
+  def test_it_can_score_letter_multipliers
+    game = Scrabble.new
+    expected = 9
+    actual = game.score_with_multipliers("hello", [1,2,1,1,1])
+
+    assert_equal expected, actual
+  end
+
 end
```

Comment:
I'd like to see you add a few more tests to double-check functionality (e.g. words of different lengths using different letters)